src/components/TaggablePhoto/TaggablePhoto.jsx

```diff
@@ -12,13 +13,23 @@ export default class TaggablePhoto extends React.Component {
   }
 
   async componentDidMount() {
-    // const { taggedUsers } = await fetch(INSERT API CALL FOR PHOTO'S TAGGED USERS HERE);
-    const value = (taggedUsers || []).map(({ email }) => email);
-    this.setState({ value });
+    const {
+      match: {
+        params: { albumId, photoId },
+      },
+    } = this.props;
+    try {
+      const taggedUsers = await fetchTaggedUsers(albumId, photoId);
+      const value = taggedUsers;
+      this.setState({ value });
```

Comment:
I'd be pro you renaming the state value to be taggedUsers since "value" is vague!